### ISSUE TITLE
[Feature Request] Add icon, buttons and extra info to player status page?

### ISSUE THREAD

**Issue by Cormak** [segler-alex/RadioDroid#252]
Hey guys. Just tried out your 0.46 beta and it's already a huge improvement. The player is starting to feel like a modern app. However, the one part of the app that needs the most polish and features is the player status page. 

1) We should have the station icon in the player status page somewhere, kind of like album art, but small. It adds some color to an otherwise monotone page. 
2) We should have extra info in the player status like session playtime so we can see how long the station been playing for that session. It would be just as useful as the amount of data used during that session.
3) We should have a share button on the player status page so we don't have to go to the listings to find the share button. Whether on the listings or the player status page, we should be able to have the same functionality. 
4) The record button should be smaller than the play/stop buttons to reduce the chances of us hitting it. I imagine more people just want to listen rather than record the audio, so the record button should be secondary and smaller than the playback functions. 

Overall, the player status page will look more like a modern audio player with these improvements, and allow the player status page to be just as functional as the listings, which it should be. 

What do you guys think?

**Comment by werman** [comment]
Yes, I agree this page should be reworked. From my pov it would be great to have timeline of station playback with song names and time codes in it with a quick way to copy, search the artist/song and view lyrics (via external app).

**Comment by Cormak** [comment]
@werman Yes! The thing I noticed as well is that the notification has the session playtime but the player status page doesn't have the session playtime at all, which seems totally inconsistent. If anything, the player status page should have all the info for that current playback session. 

Furthermore, it's very strange how the player status page doesn't have a share function but the station listing does for every station. 

I also think adding a way to tap and copy "artist - title" to clipboard would be a nice way to save the current song info for later, as a start.
